schema/__tests__/sale_artworks.test.js

```diff
@@ -186,14 +187,64 @@ describe("Sale Artworks", () => {
         }
       }
     `
-    const { sale_artworks: { aggregations } } = await execute(gravityResponse, query, {
-      aggregations: ["total", "medium", "followed_artists"],
-    })
+    const { sale_artworks: { aggregations } } = await execute(gravityResponse, query)
 
     expect(aggregations.length).toBeGreaterThan(0)
-
     aggregations.forEach(aggregation => {
       expect(aggregation.counts.length).toBeGreaterThan(0)
     })
   })
+
+  describe("authentication", () => {
+    const hits = Array(10)
+    const gravityResponse = {
+      hits,
+      aggregations: {
+        total: {
+          value: hits.length,
+        },
+      },
+    }
+    const query = gql`
```

Comment:
`gql`? I thought it was `graphql`. Gosh JavaScript moves quickly.

Reply:
It's just a [function call](https://github.com/artsy/metaphysics/blob/master/test/gql.js), basically -- can be labeled whatever. (Though its common to see `gql` in Apollo and `graphql` in Relay.)